.stylelintrc.json

```diff
@@ -58,6 +60,30 @@
         "type": "at-rule",
         "hasBlock": true
       }
+    ],
+    "scale-unlimited/declaration-strict-value": [
+      [
+        "border-radius",
+        "color",
+        "background-color",
+        "opacity",
+        "z-index",
+        "font-weight",
+        "box-shadow",
+        "font-size"
+      ],
+      {
+        "ignoreValues": [
+          "inherit",
+          "transparent",
+          "initial",
+          "unset",
+          "0",
+          "none",
+          "currentcolor",
+          "bolder"
```

Comment:
Why do we need `bolder` in the ignore values?

Reply:
You are right. We don't need it. I deleted it